### PR TITLE
add notes for n23201

### PR DESCRIPTION
**Pull request by glozow**

### CODE REVIEW

**Comment by glozow** [comment]
cc @achow101 if you have time to take a look?

File: _posts/2022-01-19-#23201.md [hunk 1]
```diff
@@ -10,14 +10,80 @@ status: upcoming
...
+
+* In some use cases, users may want to fund a transaction with *external inputs*, i.e., inputs that
+  are not controlled by the user themselves, but some external wallet such as that of a LN
+  channel counterparty. The challenge in this scenario is accounting for these inputs when funding
+  the fees of the transaction - the wallet needs to know their size.
+
+    - For standard output types, the wallet can approximate the size of the input and corresponding
+      witness data. For example,
+[this stackexchange post](https://bitcoin.stackexchange.com/questions/48279/how-big-is-the-input-of-a-p2pkh-transaction)
+      breaks down the possible sizes of a P2PKH input.
```
Comment (achow101) [inline _posts/2022-01-19-#23201.md:27]: Technically that's not true, and the wallet doesn't do this. The pubkey could be uncompressed, we wouldn't know unless we have the solving data.

File: _posts/2022-01-19-#23201.md [hunk 2]
```diff
@@ -10,14 +10,80 @@ status: upcoming
...
+
+    - Other standard output types that involve scripts (which can be arbitrary in size), such as
+      P2SH, P2WSH and P2TR, may require the wallet to know more information.
+
+    - [PR #17211](https://github.com/bitcoin/bitcoin/pull/17211) added support for allowing
+      users to provide *solving data* including public keys, scripts, and descriptors to help the
+      wallet determine the size of external input(s).
+
+    - Some sizes might differ depending on how the external wallet represents the public key and
```
Comment (achow101) [inline _posts/2022-01-19-#23201.md:36]: The way the pubkey is represented is fixed by the UTXO via a hash. It's only the signature size that is unknown.

File: _posts/2022-01-19-#23201.md (hunk 1 above)
Comment (glozow) [reply _posts/2022-01-19-#23201.md:27]: ok i'll fix this
also i should just review #17211 at some point, ngl i don't know how it works

**Comment by glozow** [comment]
thanks @achow101, added some fixups in #457

**Pull request merged**